openstack/identity/v3/registered_limits/requests.go

```diff
@@ -0,0 +1,159 @@
+package registered_limits
```

Comment:
Shouldn't the package be `registeredlimits`. I don't recall any gohpercloud package to have `_` or any other seperator. It's typically lower case and no seperator. @mandre ?